File: cilicili-go/model/entity/video.go
package entity

type Video struct {
	User          User
	Music         Music
	Id            string
	UserId        string
	Title         string
	Url           string
	OriginalSound bool //是否为原声 否则播放mp3
	MusicId       string
	Description   string
	Location      string //定位
	LikeCount     int64  //点赞数
	CommentCount  int64  //评论数
	ShareCount    int64  //分享数
	CreateAt      int64
}

type Music struct {
	Id        string
	Name      string //名字
	Author    string //作者
	count     int64  //使用该音乐人数
	PosterUrl string //音乐海报封面图片地址
	Url       string //mp3地址
	CreateAt  int64
}

type VideoLike struct {
	id      string
	videoId string
	userId  string
}

File: cilicili-cloud/cilicili-upload/src/main/java/com/gaoyuanming/upload/model/entity/Video.java
package com.gaoyuanming.upload.model.entity;

import lombok.Data;
import lombok.NoArgsConstructor;
import org.hibernate.annotations.GenericGenerator;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;
import javax.persistence.Table;

@Data
@Entity
@Table(name = "t_video")
@NoArgsConstructor
public class Video {

    @Id
    @GeneratedValue(generator = "uuid2")
    @GenericGenerator(name = "uuid2", strategy = "org.hibernate.id.UUIDGenerator")
    private String id;

    private String url;

    private String poster;

    private String userId;

    private String userNickname;

    private String userAvatar;

    private boolean useMusic;

    private String musicId;

    private String musicName;

    private String musicAuthor;

    private String musicPoster;

    private String musicUrl;

    private String title;

    private int likeCount;

    private int commentCount;

    private int shareCount;

    private boolean status;

}
